azure-pipelines.yml

```diff
@@ -13,7 +13,7 @@ variables:
 trigger:
   - master
   - release/*
-  - internal/release/*
+  - internal/release/6.*
```

Comment:
Why this change?

Reply:
I added an extra commit in an attempt to figure out why the database cleanup was failing in the release branches but succeeded (I thought) in 'master'. Since the 'master' validation build now failed, I suspect something else is going on -- perhaps LocalDb is slightly different or missing on these agents.

Reply:
Will remove that commit once I understand what's going on w/ the failing database cleanups.

Reply:
The commit adding the unrelated changes is gone now. Much less to see here 😈